README.md

```diff
@@ -1,9 +1,9 @@
 # [ECMAScript](https://github.com/tc39/ecma262) proposals
 
-* [Stage 1 Proposals](stage-1-proposals.md)
-* [Stage 0 Proposals](stage-0-proposals.md)
-* [Finished Proposals](finished-proposals.md)
-* [Inactive Proposals](inactive-proposals.md)
+- [Stage 1 Proposals](stage-1-proposals.md)
```

Comment:
Looks like my editor tried to auto-format it.

I did say save without formatting.